### DISCUSSION TITLE
Fingerprint of FaceId identifiers

### DISCUSSION THREAD

**Discussion by kwyz** [smstuebe/xamarin-fingerprint#221] [Q&A] [answered]
Thanks allot for this awesome package.
My project is based on the .Net MAUI + Blazor, witch let me make crossplatform aplication.
The returned response after successful authorization via fingerprint has only these formats
 ` {
   Authenticated: true,
   Status: Succeeded,
   ErrorMessage: null
} `
And this is not quite what I expected to receive, after successful authorization. Based on the business logic of my application, I need to aquire a fingerprint identifier or faceId identifier, to futher encrypt data on secureStore of application, to prevent application cache dumps or other  storage and cache leaks  of security.
Based on the fingerprint identifier or faceId identifier, i would know exacly that current user is the phone owner and extract and decrypt storedData, otherwise the data will remain encrypted and would not be accesed or readead by other person.

Soo the main question is that if exist an way to get those identifiers. If so, could you please provide some usefull links or web articles or other kind of stuff that will help me to implement this logic based of project configuration, .Net MAUI + Blazor.
In addition, I have done some researchs, but unfortunaly didn't find any kind of information related to project configuration. Oly the fact that I can perform this logic, but it require implemetation for each platform separately of this functionality (fingerprint identifier + encrypt data), witch is not quite what I need. 

Thanks in advance for your time and help.

**Comment by smstuebe** [comment, marked as answer]
AFAIK you can't get identifiers. Instead you can directly perform CRUD operations on keystores using the local auth (fingerprint/face id/...). And if for example the fingerprint is deleted, the entries get invalid/unusable automatically. But this is knowledge from a few years ago. I would have to read more about it and enhance the API to support this.
